package.json

```diff
@@ -4,18 +4,50 @@
   "version": "0.0.0",
   "type": "module",
   "scripts": {
-    "dev": "vite",
+    "dev": "vite --open --host",
+    "preview": "vite preview",
     "build": "vite build",
-    "preview": "vite preview"
+    "test": "react-scripts test"
```

Comment:
- [ ] The error still persists when I try to test your application. Kindly fix that

![image](https://user-images.githubusercontent.com/98453979/212343600-7ee142d1-dcc6-42ac-a5ac-b3b3eec87399.png)

Reply:
Hello @frank1738 ,
Kindly re-install react scripts, I hope that solves the problem at your end.  (npm install react-scripts --save)
The test cases run well at my end even as I clone the project again.
![image](https://user-images.githubusercontent.com/33840476/212352592-f41f27bd-4756-4d40-a8f3-f102ffa3a86e.png)

Reply:
@SekoViper  Okey. Solved. Kindly resubmit your project for approval